src/MetadataUtility/Models/Sensor.cs

```diff
@@ -63,5 +63,10 @@ public record Sensor
         /// configuration file.
         /// </summary>
         public string[] Configuration { get; init; }
+
+        /// <summary>
+        /// Gets the temperature from the sensor.
+        /// </summary>
+        public string Temperature { get; init; }
```

Comment:
Again, document units, and make sure we're reporting units a SI units (degrees celsius)